src/Sdk/WebApi/WebApi/ResultsHttpClient.cs

```diff
@@ -234,6 +234,7 @@ private async Task UploadBlockFileAsync(string url, string blobStorageType, File
                 if (blobStorageType == BlobStorageTypes.AzureBlobStorage)
                 {
                     request.Content.Headers.Add(Constants.AzureBlobTypeHeader, Constants.AzureBlockBlob);
+                    request.Content.Headers.Add("Content-Type", "text/plain");
```

Comment:
Do we need to update the SDK path?  Or that's handled automatically for us?  In that case, once we rollout the SDK change, does this become a none issue?

Reply:
Added to the sdk case, thank you!